sum.go

```diff
@@ -7,10 +7,10 @@ import (
 	"errors"
 	"fmt"
 
-	keccak "github.com/ethereum/go-ethereum/crypto/sha3"
 	blake2b "golang.org/x/crypto/blake2b"
 	blake2s "golang.org/x/crypto/blake2s"
 	sha3 "golang.org/x/crypto/sha3"
+	keccak "leb.io/hashland/keccakpg"
```

Comment:
I'll need to vet this lib vs the ethereum one. Also it will need to be vendored with gx